Tests/SwiftKueryORMTests/TestFind.swift

```diff
@@ -113,4 +118,139 @@ class TestFind: XCTestCase {
+                  XCTAssertEqual(array.count, 1, "Find Failed: \(String(describing: array.count)) is not equal to 1")
+                }
+                expectation.fulfill()
+            }
+        })
+    }
+
+    /**
+      Testing that the correct SQL Query is created to retrieve a specific model.
+      Testing that correct amount of models are retrieved
+      Testing that models are sorted by age in descending order
+    */
+    func testFindAllOrderByDescending() {
+        let connection: TestConnection = createConnection(.returnThreeRowsSortedDescending)
+        Database.default = Database(single: connection)
+        var column: Column?
+        do {
+            let table = try Person.getTable()
+            column = table.columns.first(where: {$0.name == "age"})
+            XCTAssertNotNil(column, "Find Failed: column not found for ordering")
+            guard column != nil else {
+                return
+            }
+        } catch let error {
+            XCTAssertNil(error, "Find Failed: \(String(describing: error))")
+        }
+        performTest(asyncTasks: { expectation in
+            Person.findAll(orderBy: OrderBy.DESC(column!)) { array, error in
+                XCTAssertNil(error, "Find Failed: \(String(describing: error))")
+                XCTAssertNotNil(connection.query, "Find Failed: Query is nil")
+                if let query = connection.query {
+                  let expectedQuery = "SELECT * FROM People ORDER BY People.age DESC"
+                  let resultQuery = connection.descriptionOf(query: query)
+                  XCTAssertEqual(resultQuery, expectedQuery, "Find Failed: Invalid query")
+                }
+                XCTAssertNotNil(array, "Find Failed: No array of models returned")
+                if let array = array {
+                  for (index, person) in array.enumerated() {
+                    if index + 1 < array.count {
+                      XCTAssertGreaterThanOrEqual(person.age, array[index + 1].age, "Find Failed: Age of person: \(String(describing: person.age)) is not greater than or equal to age of next person: \(String(describing: array[index + 1].age))")
+                    }
+                  }
+                  XCTAssertEqual(array.count, 3, "Find Failed: \(String(describing: array.count)) is not equal to 3")
+                }
+                expectation.fulfill()
+            }
+        })
+    }
+
+    /**
+      Testing that the correct SQL Query is created to retrieve a specific model.
+      Testing that correct amount of models are retrieved
+      Testing that models are sorted by age in ascending order
+    */
+    func testFindAllOrderByAscending() {
+        let connection: TestConnection = createConnection(.returnThreeRowsSortedAscending)
+        Database.default = Database(single: connection)
+        var column: Column?
+        do {
+            let table = try Person.getTable()
+            column = table.columns.first(where: {$0.name == "age"})
```

Comment:
@EnriqueL8 Would this be enough for a verification if the column exists? Since the `orderBy` parameter is an OrderBy buildable that doesn't accept just any string, verification would have to be done outside of `findAll` function

Reply:
Annoyingly yes - the `OrderBy` only accepts a `Field` type that can be for example a Column but also a [RawField](https://github.com/IBM-Swift/Swift-Kuery/blob/master/Sources/SwiftKuery/RawField.swift). 
We could use [Order](https://github.com/IBM-Swift/KituraContracts/blob/master/Sources/KituraContracts/Contracts.swift#L898) that accepts a *String* and then find a *Column* matching that *String* and then create the `OrderBy` case.